R/install-dev.sh

```diff
@@ -38,7 +38,12 @@ pushd $FWDIR > /dev/null
 if [ ! -z "$R_HOME" ]
   then
     R_SCRIPT_PATH="$R_HOME/bin"
-   else
+  else
+    # if system wide R_HOME is not found, then exit
+    if ! [ `command -v R` ]; then
```

Comment:
Is it not `[ !` instead of `! [` -- were you able to test this? I doubt the integration tests have a path where R doesn't exist.